src/components/application_manager/test/commands/hmi/simple_response_from_hmi_test.cc

```diff
@@ -244,7 +252,7 @@ TEST_F(OtherResponseFromHMICommandsTest, VIGetVehicleTypeResponse_Run_SUCCESS) {
       CreateCommand<commands::VIGetVehicleTypeResponse>(command_msg));
 
   application_manager_test::MockHMICapabilities hmi_capabilities;
-  EXPECT_CALL(mock_app_manager_, hmi_capabilities())
+  EXPECT_CALL(app_mngr_, hmi_capabilities())
```

Comment:
@wolfylambova22, did you try to compile SDL with turned on `am_commands_test` target?